tests/tt_eager/python_api_testing/unit_testing/backward_ops/test_backward_remainder.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import pytest
+import ttnn
+from tests.tt_eager.python_api_testing.unit_testing.backward_ops.utility_funcs import compare_pcc, data_gen_with_range
```

Comment:
- Move the test file to TTNN and 
- Update to `tests.ttnn.unit_tests.operations.backward.utility_funcs`